src/main.ts

```diff
@@ -130,12 +131,38 @@ async function resolveConfigs(
     logger.error(chalk.red('You must provide at least 1 prompt'));
     process.exit(1);
   }
+
+  // Convert providers to array if not already an array
+  if (config.providers && typeof config.providers === 'string') {
+    config.providers = [config.providers];
+  }
   if (!config.providers || config.providers.length === 0) {
-    logger.error(
-      chalk.red('You must specify at least 1 provider (for example, openai:gpt-3.5-turbo)'),
-    );
+    logger.error(chalk.red('You must specify at least 1 provider (for example, openai:gpt-4o)'));
     process.exit(1);
   }
+  if (Array.isArray(config.providers)) {
```

Comment:
in what cases is providers not an array?

Reply:
You're right. Even with `providers: openai:gpt-3.5-turbo` it gets coerced into an array above this point. I need to narrow the type if I don't do this.